Makefile

```diff
@@ -119,6 +148,22 @@ st: run-etcd dist/calicoctl docker calico_test/.calico_test.created busybox.tar
 	           calico/test \
 	           sh -c 'cp -ra tests/st/* /tests/st && cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
 
+## Run the STs in a container using etcd with SSL certificate/key/CA verification.
+st-ssl: run-etcd-ssl dist/calicoctl docker calico_test/.calico_test.created busybox.tar calico-node.tar routereflector.tar
+	docker run --uts=host \
+	           --pid=host \
+	           --net=host \
+	           --privileged \
+	           -e HOST_CHECKOUT_DIR=$(HOST_CHECKOUT_DIR) \
+	           -e ETCD_SECURITY_TYPE=ssl \
+	           -e HOST_CERT_DIR=`pwd`/certs \
```

Comment:
Why do you want to use the same directory hierarchy in the container?  (i.e. why the 'pwd' here?)